### ISSUE TITLE
Multiple pdf views in integrated pdf viewer

### ISSUE THREAD

**Issue by CarlosContrerasQ12** [texstudio-org/texstudio#3145]
Sometimes it is useful to see other pdf document (not the compiled one) while you are writing. However, right now, the only way I've found to work together with the compiled pdf and the other pdf is to open multiple pdf viewer windows that are no integrated to the editor. It would be useful that texstudio had tab manager in the integrated pdfviewer that allows users to interchange between multiple pdf, not necessarily the compiled one.

**Comment by zhong-yy** [comment]
Agree! Sometimes we need edit and compile documents, such as one manuscript, one supplement and one response file to reviewers.
